src/libsyncengine/propagation/operation_sorter/operationsorterworker.cpp

```diff
@@ -287,7 +287,7 @@ void OperationSorterWorker::fixMoveBeforeDelete() {
             }
 
             SyncPath sourcePath = moveOp->affectedNode()->moveOrigin().value();
-            if (Utility::startsWith(sourcePath.lexically_normal(),
+            if (Utility::isSameOrChildPath(sourcePath.lexically_normal(),
```

Comment:
The word "child" often refers to a child node in the sense of the graph terminology, when the graph is a tree. 
The word "descendant" seems more accurate to me since it applies to nodes which can be lower in the tree hierarchy with respect to a reference root.

Reply:
Thank you for the clarification. Here is the modification: [7802b0a](https://github.com/Infomaniak/desktop-kDrive/pull/356/commits/7802b0a0e8ee0a2cafc607c03d623584df9138b8)